### PR TITLE
Add new snippets for rspec

### PR DESCRIPTION
**Pull request by ToruIwashita**

### CODE REVIEW

**Comment by lpil** [comment]
I don't like how short the triggers for these snippets are.

File: snippets/ruby.snippets [hunk 1]
```diff
@@ -723,3 +723,11 @@ snippet is
 	it { should ${0} }
 snippet isn
 	it { should_not ${0} }
+snippet ie
+	it { expect(${1:object}).${0} ${1} }
```
Comment (lpil) [inline snippets/ruby.snippets]: Would `it { expect(${1:object}).${1} ${0} }` not be better?

**Comment by ToruIwashita** [comment]
I understand.
Well, How about this?
```
snippet iexp
  it { expect(${1:object}).${1} ${0} }
snippet iexpb
  it { expect { ${1:object} }.${1} ${0} }
snippet iiexp
  it { is_expected.to ${0} }
snippet iiexpn
  it { is_expected.not_to ${0} }
```

**Comment by lpil** [comment]
Much :)

**Comment by ToruIwashita** [comment]
Thanks! Updated.

**Comment by lpil** [comment]
Perfect!

**Pull request merged**